_bioinfcomputing/access_methods.md

```diff
@@ -1,5 +1,210 @@
 ---
 title: Computing Resource Access Methods
-last_modified_at: 2019-04-01
-primary_reviewers:
+last_modified_at: 2019-04-11
+primary_reviewers: bmcgough
 ---
+
+## Desktop Computing and Mobile Devices
+
+Client devices are supported by Center IT (ADM, CRD, VIDD) as well as Division IT groups (BSD, HB, PHS).  Please see Center IT's page in CenterNet about [Laptops and Desktops](https://centernet.fredhutch.org/cn/u/center-it/help-desk/laptops-and-desktops.html) for more information about support for desktop computing.
+
+>Note: If you request a Linux Desktop, Center IT will recommend that you use NoMachine from a Windows or Mac system instead as this is the most appropriate choice for most users but not all. If it is required for your work, it can be supported.  
+
+Please see Center IT pages for more information on [Mobile Device Services](https://centernet.fredhutch.org/cn/u/center-it/services/mobile_device_service.html) or contact [Help Desk](https://centernet.fredhutch.org/cn/u/center-it/help-desk.html) with questions or problems.  
+
+
+## SSH Clients for Remote Computing Resources
+
+We use two common protocols to access remote compute resources. The first is the Hypertext Transfer Protocol (HTTP). This is the way your browser communicates with remote web servers. The second common protocol is Secure SHell (SSH). This is the method used to run a program on a remote server. Typically the program run is a `shell` that gives you an interactive command line on the remote host.
+
+Many programs you will want to run on a remote compute server are command line programs. These are easily executed from your `shell` remote process. You use this method when opening a terminal (`shell`) on a compute server (like `rhino`) and typing the name of the program or script.
```

Comment:
Shouldn't this be worded more like, "You use this method when opening a terminal (shell) on your computer and then connecting to a remote compute server (like rhino) and typing the name of the program or script.  ????